docs/evaluate/temporal-cloud/gcp-private-service-connect.mdx

```diff
@@ -17,4 +17,80 @@ tags:
   - private service connect
 ---
 
-Awaiting draft.
\ No newline at end of file
+#### GCP Private Service Connect
+
+[GCP Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) allows you to open a path to Temporal without opening a public egress.
+It establishes a private connection between your Google Virtual Private Cloud (VPC) and Temporal Cloud.
+This one-way connection means Temporal cannot establish a connection back to your service.
+This is useful if normally you block traffic egress as part of your security protocols.
+If you use a private environment that does not allow external connectivity, you will remain isolated.
+
+Set up Private Service Connect with Temporal Cloud with these steps:
+
+1. Open the Google Cloud console
+2. Navigate to _Network Services_, then _Private Service Connect_. If you haven't used _Network Services_ recently, you might have to find it by clicking on _View All Products_ at the bottom of the left sidebar.
+
+   ![GCP console showing Network Services, and the View All Products button](/img/gcp-private-service-connect/gcp-console.png)
```

Comment:
Looking at the Vercel rendering, this needs to be sized since it's a vertical screenshot

Reply:
@MasonEgger do you know what is done generally in this case? do we directly use a `<img>` HTML block for sizing?